scripts/run_app.sh

```diff
@@ -44,11 +51,11 @@ if [ ! -s "app/static" ]; then
   npm run compile
 fi
 
-url="`python token_generator.py`"
+url="`run_python token_generator.py`"
 
-if [ -z "${TRAVIS}" ]; then
+if [ -z "${TRAVIS}"] || [-z "${DOCKER}"]; then
   open_url $url &
```

Comment:
You need to move this command into the docker conditional. 
As it tries to launch chrome in the container.